questions/qa-address-formats.en.md

```diff
@@ -0,0 +1,131 @@
+
+### Key differences in address formats
+
+#### **Street numbers and house numbering**
+
+**Order of street name and house number**
+
+In many countries like the United States, United Kingdom, and France, the house number precedes the street name (e.g., "123 Main Street"). However, in Germany and Switzerland, the street name comes first, followed by the house number:
+
+```
+Sternengasse 3
+35108 Allendorf
+Deutschland
+```
+
+This difference has important implications for form design, as users from these countries may expect to enter their address in this order.
+
+**Complex addressing systems**
+
+In Colombia, instead of numbering houses sequentially along a street, addresses are based on the distance from the nearest intersecting street. For example, `Calle 122 # 18 – 15` means 122nd street, 15 metres from the intersection with 18th street.
+
+Kyoto uses a similar system based around street intersections, with cardinal directions indicating the location relative to the intersection. For instance, the address of Kyoto Tower is:
+
+```
+京都市下京区 Kyōto-shi, Shimogyō-ku
+烏丸七条下ル Karasuma-Shichijō-sagaru
+東塩小路町 721-1 Higashi-Shiokōji 721-1
+```
+
+This second line means "south of the intersection of Karasuma and Shichijō streets".
+
+Elsewhere in Japan, street names are often not used at all. Instead, buildings are referenced by block numbers, and their numbering may not follow geographic order (e.g., they can be the order in which buildings were constructed). For example, in the following address:
+
+```
+305-0051, Ibaraki, Tsukuba, Ninomiya 1-chōme, 6-13
+```
+
+305-0051 is the postal code, Ibaraki is the prefecture, Tsukuba is the city, Ninomiya 1-chōme is the district, 6 is the block number, and 13 is the building number.
+
+#### **Order of address components**
+
+While most countries recommend that users format addresses in order from the most detailed part (such as street and house number) to the most general part (like city and province), there are some cases that slightly deviate from the general rule.
+
+In China and Japan, addresses are written starting with the postal code, followed by the largest administrative area (e.g., province), and down to the smallest unit (e.g., room number). For example, here is an example of a Chinese address:
+
+```
+100083 北京市 海淀区 学院路 37号
+```
+
+(Postal code → City → District → Street → House number).
+
+Russia often follow a "largest-to-smallest" order, starting with the country. For example:
+
+```
+Russia, Moscow, Tverskaya Street, 16
+```
+
+However, Russia also uses the "smallest-to-largest" order commonly seen internationally, so developers should account for both formats.
+
+#### **Countries without postal codes**
```

Comment:
I'd move this section below Postal Code Systems Aroudn the World

Reply:
Done.